spec/features/users/show_spec.rb

```diff
@@ -0,0 +1,74 @@
+require "rails_helper"
+
+RSpec.describe "as a visitor" do
+  describe "when I click on a user's name for any book review" do
+    before :each do
+      @book_1 = Book.create!(title: "The Frozen Deep", page_count: 106, year_published: 1874, thumbnail: "https://images.gr-assets.com/books/1328728986l/1009218.jpg")
+      @book_2 = Book.create!(title: "To Kill a Mockingbird", page_count: 281, year_published: 1960, thumbnail: "https://upload.wikimedia.org/wikipedia/en/7/79/To_Kill_a_Mockingbird.JPG")
+      author_1 = @book_1.authors.create!(name: "Wilkie Collins")
+      author_2 = @book_1.authors.create!(name: "Charles Dickens")
+      author_3 = @book_2.authors.create!(name: "Harper Lee")
+      @user_1 = User.create(username: "Chris Davis")
+      @user_2 = User.create(username: "Alexandra Chakeres")
+      @review_1 = @book_1.reviews.create!(title: "This book rocks!", rating: 5, text: "Read it!", user: @user_1)
+      @review_2 = @book_2.reviews.create!(title: "This book sucks!", rating: 1, text: "Don't read it!", user: @user_1)
+    end
+
+    it "i am taken to a show page for that user." do
+      visit book_path(@book_1)
+
+      click_link("#{@user_1.username}")
```

Comment:
```suggestion
      click_link(@user_1.username)
```

Reply:
String is needed for click_link, changed to a to_s call.